src/main/java/com/afornalik/ox/view/UIOperations.java

```diff
@@ -0,0 +1,17 @@
+package com.afornalik.ox.view;
+
+import com.afornalik.ox.model.board.Board;
+
+public interface UIOperations {
+
+    void print(String draw);
+
+    void drawBoard();
+
+    void drawBoard(Board board);
```

Comment:
Why two options? Maybe just leave one of them and stick to it.

Reply:
Removed